example/lib/main.dart

```diff
@@ -32,10 +32,15 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> {
+  final omisePayment = OmisePayment(publicKey: "pkey", enableDebug: false);
```

Comment:
Does this mean that debug will be enabled by default?

Reply:
no its false by default.